### DISCUSSION TITLE
Best practice to replicate the footer?

### DISCUSSION THREAD

**Discussion by tracknftlive** [cloudscape-design/components#274] [Q&A] [answered]
Hi,

What is the best practice to replicate the following footer:
<img width="1263" alt="image" src="https://user-images.githubusercontent.com/111466576/190260082-2a8b8635-2a44-43c5-b94b-9e6819906ebc.png">

I see that contentHeader is being deprecated for the contentLayout, is the footer going to be there as well?
What is the actual best way to have the same footer as you? I don't see any footer example on the doc yet?

**Comment by connorlanigan** [comment, marked as answer]
There is currently no officially supported component or method for this included in Cloudscape. I recommend that you implement this as a custom component.
